app/models/fine.rb

```diff
@@ -77,10 +77,21 @@ def bill_date
     Time.zone.parse(fields['billDate']) if fields['billDate']
   end
 
+  def days_overdue
+    return unless bill_date
+    return unless bill_date.past?
+
+    ((Time.zone.now - bill_date).to_i / 60 / 60 / 24).round + 1
```

Comment:
I think this (+ 1) is the correct logic as illustrated by the test. I believe if the bill_date is today's date, the item is already 1 day overdue (and not 0 days overdue)?